### PR TITLE
Feat/runs report page

### PR DESCRIPTION
**Pull request by raylrui**
initial PR for  report page:
1.  page content ( still in planing/development ) until requirement is finalised
2. html and pdf export function to report
3. refactor some api for stats api
4. refactor icon button to be a new component
5. new barchart for data display
6. refactor/remove some mock data and storybooks..
7. fix the Security - Dependabot alerts (update eslint)

couple with api PR: https://github.com/umccr/orcabus/pull/725

For report page process (planing)
1. enter in runs/workflow page
image

8. selection time range:
image

9. report content (preview):
image

10. export html and pdf format
image

### CODE REVIEW

File: src/api/types/metadata.d.ts [hunk 1]
```diff
@@ -1023,10 +1023,6 @@ export interface operations {
             query?: {
                 assay?: string | null;
                 coverage?: number | null;
-                /** @description Filter based on 'coverage' that is greater than or equal to the given value. */
```
Comment (williamputraintan) [inline src/api/types/metadata.d.ts:1026]: Perhaps this is outdated with my recent commit, these fields should exist in the current API.

Comment (raylrui) [reply src/api/types/metadata.d.ts:1026]: I just generate again with latest orcabus code, it seems no changes, could you have a check again?

**Comment by raylrui** [comment]
Will refactor this to do the stats calculate in the api side, and implement pagination for workflow list.
For export func, we may have a another sync function to read all data from db.

Will implement limitation warning like over 1000 records from api side if request over 1000 records when do the status.

**Comment by raylrui** [comment]
closed, new deisgn wil be make

**Pull request closed without merging**